Main.gd

```diff
@@ -0,0 +1,54 @@
+extends Node
+
+var rods # all the radioactive rods in the scene
```

Comment:
To help move this towards a more general case, why don't we replace `rods` with `sources` and `radiation_array` with `radiation_map`?